### PR TITLE
4296 If trial location is not selected and rqt hasn't been uploaded show order for requested trail location

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

**Review by alsmith1 — APPROVED** [review]

File: web-client/src/presenter/computeds/startCaseInternalHelper.test.js [hunk 1]
```diff
@@ -278,4 +278,15 @@ describe('case detail edit computed', () => {
     });
     expect(result.showOrderForFilingFee).toBeFalsy();
   });
+
+  it('sets showOrderForRequestedTrialLocation true if Requested Trial Location is specified', () => {
```
Comment (matthopson) [inline web-client/src/presenter/computeds/startCaseInternalHelper.test.js]: In the helper it's setting it to false if `form.preferredTrialCity` is set.

**Review by sutt0n — APPROVED** [review]

**Comment by matthopson** [comment]
DAT BRANCH NAME.
`git checkout` just can't handle it.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/computeds/startCaseInternalHelper.test.js (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/presenter/computeds/startCaseInternalHelper.test.js]: My bad, it's been fixed.

**Comment by ericsorenson** [comment]
> DAT BRANCH NAME.
> `git checkout` just can't handle it.

https://desktop.github.com/ :-D

**Review by matthopson — APPROVED** [review]

**Pull request merged**